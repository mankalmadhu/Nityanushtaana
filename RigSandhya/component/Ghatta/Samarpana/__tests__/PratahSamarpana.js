import React from 'react';
import ಪ್ರಾತಸ್ಸಮರ್ಪಣ from '../PratahSamarpana';
import Renderer from 'react-test-renderer';

jest.mock('../../Ghatta.vinyasa');
jest.mock('../../../Vishaya/Vishaya',()=>'ವಿಷಯ');

test("ಪ್ರಾತಸ್ಸಮರ್ಪಣ", () => {
   expect(Renderer.create(
    <ಪ್ರಾತಸ್ಸಮರ್ಪಣ />
  ).toJSON()).toMatchSnapshot();

});