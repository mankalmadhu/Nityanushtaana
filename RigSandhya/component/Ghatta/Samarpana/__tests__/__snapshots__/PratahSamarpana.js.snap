// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ಪ್ರಾತಸ್ಸಮರ್ಪಣ 1`] = `
<ವಿಷಯ
  ವಿನ್ಯಾಸ="ವಿನಿಯೋಗ"
>
  ಅನೇನ ಪ್ರಾತಸ್ಸ೦ಧ್ಯಾವ೦ದನೇನ ಭಗವಾನ್ ಸರ್ವಾತ್ಮಕಃ ಶ್ರೀ ತೇಜೋ ಲಕ್ಷ್ಮೀನೃಸಿ೦ಹ ಸ್ವಾಮಿನಃ ಪ್ರೀಣಾತು |
</ವಿಷಯ>
`;
